config/samples/certificate.yaml

```diff
@@ -10,7 +10,7 @@ kind: Certificate
 metadata:
   name: harbor-sample-public
 spec:
-  secretName: public-certificate
+  secretName: harbor-sample-certificate
```

Comment:
The goal of this Certificate resource is to generate the [`tlsSecretName` of harbor sample](https://github.com/goharbor/harbor-operator/blob/master/config/samples/goharbor_v1alpha1_harbor.yaml#L7).  
[./config/samples/goharbor_v1alpha1_harbor.yaml](https://github.com/goharbor/harbor-operator/blob/master/config/samples/goharbor_v1alpha1_harbor.yaml#L7) needs to be updated too.  
Such as other reference in documentation [./docs/installation.md](https://github.com/goharbor/harbor-operator/blob/master/docs/installation.md#deploy-the-sample)